src/Command/Core/CoreClearCommand.php

```diff
@@ -41,11 +41,22 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $application = $this->getApplication();
-        /** @var \Gush\Application $application */
-        $config = $application->getConfig();
-        (new Filesystem())->remove($config->get('cache-dir'));
+        $cacheDir = $this->getConfig()->get('cache-dir');
 
-        $output->writeln('Cache has been cleared.');
+        // Confirm is to ensure the Cache-directory is what the user expects
```

Comment:
It makes sure the user knowns which directory will be deleted, if you mis-configured the cache-dir to say /tmp you are gonna have a really bad time! As the cache configuration is handled using the Gush-home or ENV variable we should document somewhere how to change the cache-dir, and make it clear the directory should be stand-alone like `/tmp/gush` and not `/tmp`.

*It was actually done to safely manually test this command :) I don't want that a misconfiguration deletes half my disks content!*